Fix few issues in learn pages (#143)
ballerina-platform/ballerina-dev-website: learn/how-to-test-ballerina-code.md

`after: "<function name>"`: Name of the function to be run just after the test is run.
 

> `dependsOn: ["<function names>", …]`: A list of function names the test function depends on, and will be run before the test. The list of functions provided has no order of execution. The current test function will depend on the list provided and that list will run in whatever order, the order in which the comma-separated list appears has no prominence. In case there needs to be an order, the way to do that is to define a sequence of test functions with one point to another based on dependency using `dependsOn` parameter in each one's config.

Reviewer: ```suggestion
`dependsOn: ["<function names>", …]`: A list of function names on which the test function depends. These will be run before the test. The list of functions provided has no order of execution. The current test function will depend on the list provided and that list will run in whatever order. Thus, the order in which the comma-separated list appears has no prominence. In case there needs to be an order, define a sequence of test functions with one pointing to another based on dependency using the `dependsOn` parameter in each one's config.
```
